src/licensedcode/data/licenses/any-osi.LICENSE

```diff
@@ -0,0 +1,16 @@
+---
+key: any-osi
+short_name: Any OSI License
+name: Any OSI License
+category: Unstated License
+owner: Unstated
+spdx_license_key: any-OSI
+other_urls:
+    - https://metacpan.org/pod/Exporter::Tidy#LICENSE
+ignorable_urls:
+    - http://www.opensource.org/licenses/alphabetical
+---
+
+{{Pick your favourite OSI approved license}} :)
```

Comment:
We cannot and should not use required phrases in licenses as the text is designed to be reused as reference. I would remove the curly braces and use instead a `minimum_coverage: 100` to ensure this is all matched. 
You can also add a `{{Pick your favourite OSI approved license}}` text rule if you want, though this will likely be misleading, as from what I can see in the wild, use of this sentence solo is done only when OSI license(s) have been picked.

Reply:
Updating, thanks!